sdk/webpubsub/Microsoft.Azure.WebPubSub.Common/api/Microsoft.Azure.WebPubSub.Common.netstandard2.0.cs

```diff
@@ -4,15 +4,20 @@ public sealed partial class ConnectedEventRequest : Microsoft.Azure.WebPubSub.Co
     {
         public ConnectedEventRequest(Microsoft.Azure.WebPubSub.Common.WebPubSubConnectionContext context) : base (default(Microsoft.Azure.WebPubSub.Common.WebPubSubConnectionContext)) { }
     }
+    [System.Runtime.Serialization.DataContractAttribute]
     public sealed partial class ConnectEventRequest : Microsoft.Azure.WebPubSub.Common.WebPubSubEventRequest
     {
         public ConnectEventRequest(Microsoft.Azure.WebPubSub.Common.WebPubSubConnectionContext context, System.Collections.Generic.IReadOnlyDictionary<string, string[]> claims, System.Collections.Generic.IReadOnlyDictionary<string, string[]> query, System.Collections.Generic.IEnumerable<string> subprotocols, System.Collections.Generic.IEnumerable<Microsoft.Azure.WebPubSub.Common.WebPubSubClientCertificate> certificates) : base (default(Microsoft.Azure.WebPubSub.Common.WebPubSubConnectionContext)) { }
+        [System.Runtime.Serialization.DataMemberAttribute(Name="claims")]
         [System.Text.Json.Serialization.JsonPropertyNameAttribute("claims")]
         public System.Collections.Generic.IReadOnlyDictionary<string, string[]> Claims { get { throw null; } }
```

Comment:
It's really cool that this ended up working.  I think it's way less likely to break if you keep the serialization tied directly to the type.